common/djangoapps/student/tests/tests.py

```diff
@@ -487,6 +484,31 @@ def test_linked_in_add_to_profile_btn_with_certificate(self):
         )
         self.assertContains(response, expected_url)
 
+    @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+    def test_dashboard_metadata_caching(self):
+        """
+        Check that the student dashboard makes use of course metadata caching.
+
+        The first time the student dashboard displays a specific course, it will
+        make a call to the module store. After that first request, though, the
+        course's metadata should be cached as a CourseOverview.
+        """
+        # Log in and enroll the user in a course.
+        self.client.login(username="jack", password="test")
+        CourseEnrollment.enroll(self.user, self.course.id)
+
+        # The first request will result in a modulestore query.
+        response_1 = self.client.get(reverse('dashboard'))
```

Comment:
you should wrap this call in a `check_mongo_calls` as well. This way, we are certain that your subsequent call to `check_mongo_calls` is correct in actually reducing the number of calls.